lib/chrome-out.js

```diff
@@ -40,7 +40,15 @@ module.exports = function chromeOut (log, options, Runtime) {
 
     for (let arg of args) {
       if (arg.type === 'string') {
-        data.push(arg.value);
+        let value = arg.value;
+        if ('win32' == process.platform) {
+          value = value.replace("âœ“", '\u221A');
+          value = value.replace("✓", '\u221A');
+        }
+        else {
+          value = value.replace("âœ“", '\u2713');
+        }
```

Comment:
Is this even a problem on non-windows platforms?

Reply:
Yes, on my Ubuntu machine (it may be running 18.04?) I still received this garbled text. I suspect this may be an underlying content-encoding issue, but in the meantime this should help resolve the primary case -- showing a checkmark on passed tests.

I omitted anything for X marks, because Mocha does not appear to use them (though perhaps it depends on the template?)

The Windows code here has an extra line to also replace proper checkmarks with the square root mark instead, so you do not get a question mark box instead.

Reply:
> Yes, on my Ubuntu machine (it may be running 18.04?)

Are you sure it's not just your terminal setup wrong? Here I am using mocha-chrome [in travis CI on Ubuntu Trusty](https://travis-ci.org/uber/cadence-web/builds/446404783). Also no issues on OS X.

I'm not a fan of hacky fixes for windows without fully researching the root cause.

Reply:
This is an Ubuntu standard install on a desktop machine. I have not modified my terminal.

The issue appears in CircleCI for our project as well:

https://circleci.com/gh/openfl/openfl/11192

The Windows-specific replacement occurs in Mocha when outputting directly to a command-prompt:

https://github.com/mochajs/mocha/blob/d8157d0a2eca0b1c23564baca33352bf4442decb/lib/reporters/base.js#L75-L86

When running through a headless browser, it makes sense to format the web-based output for the command-prompt. For Windows, that does require a replacement